lms/product/canvas/product.py

```diff
@@ -19,3 +19,6 @@ class Canvas(Product):
     plugin_config: PluginConfig = PluginConfig(grouping=CanvasGroupingPlugin)
 
     settings_key = "canvas"
+
+    use_grading_bar = False
```

Comment:
This is a product wide decision for now. I reckon it belongs here at the top level.